src/views/confirm-identity-verification/confirm-identity-verification.njk

```diff
@@ -10,6 +10,15 @@
 
 {% include "partials/user_name.njk" %}
 <h1 class="govuk-heading-l">{{ i18n.confirmVerificationHeading }}</h1>
+
+<div class="govuk-warning-text">
+    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+    <strong class="govuk-warning-text__text">
+      <span class="govuk-visually-hidden">Warning</span>
```

Comment:
This warning will need translating as it is read aloud by screen readers

Reply:
Added text to common.json locales file to account for Welsh translation